Gems/ONNX/Code/Source/Clients/Mnist.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/IO/SystemFile.h>
+
+#include <ONNX/Model.h>
+#include "upng/upng.h"
+
+namespace Mnist
+{
+    //! Holds the digit that was inferenced and the time taken for a single inference run.
+    //! Only used by MnistExample().
+    struct MnistReturnValues
+    {
+        int64_t m_inference;
+        float m_runtime;
+    };
+
+    //! Holds the data gathered from RunMnistSuite(), which tests the MNIST ONNX model against images from the MNIST dataset.
+    struct InferenceData
+    {
+        float m_totalRuntimeInMs;
+        float m_averageRuntimeInMs;
+        int m_totalNumberOfInferences;
+        int64_t m_numberOfCorrectInferences;
+    };
```

Comment:
It would be safer to initialize these arithmetics to a known value.

Reply:
Changed